test/integration/zero-governor/set-cash-token/setCashToken.t.sol

```diff
@@ -2,7 +2,10 @@
 
 pragma solidity 0.8.23;
 
-import { IntegrationBaseSetup, IBatchGovernor, IGovernor, IZeroGovernor } from "../../IntegrationBaseSetup.t.sol";
+import { IGovernor } from "../../../../src/abstract/interfaces/IGovernor.sol";
+import { IBatchGovernor } from "../../../../src/abstract/interfaces/IBatchGovernor.sol";
+
+import { IntegrationBaseSetup, IZeroGovernor } from "../../IntegrationBaseSetup.t.sol";
```

Comment:
same here?

Reply:
Fixed the import paths in this commit: https://github.com/MZero-Labs/ttg/pull/263/commits/6cf80ea42e617e75069247b0dfbb9cfa871ecb80